next/src/components/_global/sliderComponent/swiperComponent.tsx

```diff
@@ -0,0 +1,125 @@
+'use client';
+
+import 'swiper/css';
+import { useRef, useState } from 'react';
+import { Swiper, SwiperSlide } from 'swiper/react';
+import { A11y } from 'swiper/modules';
+import styles from './sliderComponent.module.scss';
+
+export default function SwiperComponent({ children, length }: { children: React.ReactNode[]; length: number }) {
+  const [activeIndex, setActiveIndex] = useState(0);
+
+  const swiperRef = useRef(null);
+
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-call, @typescript-eslint/no-unsafe-return, @typescript-eslint/no-unsafe-member-access
+  const handlePrev = () => swiperRef.current?.swiper?.slidePrev();
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-call, @typescript-eslint/no-unsafe-return, @typescript-eslint/no-unsafe-member-access
+  const handleNext = () => swiperRef.current?.swiper?.slideNext();
+
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-return, @typescript-eslint/no-unsafe-call, @typescript-eslint/no-unsafe-member-access
```

Comment:
Generally you shouldn't ignoring the eslint and typescript's errors. Please make it works without need to disabling it with commands.